pkg/controllers/openshift-controller-manager.go

```diff
@@ -79,6 +80,12 @@ func (s *OCPControllerManager) configure(cfg *config.MicroshiftConfig) error {
 	s.ConfigFilePath = options.ConfigFilePath
 	s.Output = options.Output
 
+	if err := assets.ApplyNamespaces([]string{
+		"assets/core/0000_50_cluster-openshift-controller-manager_00_namespace.yaml",
+	}, cfg.DataDir+"/resources/kubeadmin/kubeconfig"); err != nil {
+		logrus.Warningf("failed to apply openshift namespaces %v", err)
```

Comment:
logrus.Warningf("Failed to apply openshift namespaces %v", err)

Reply:
looking thru the code, the other examples show lower case, i wasn't sure so i went w/ that

Reply:
I know that is why I didn't request review changes, the klog PR change all the lower case to upper case as per https://pkg.go.dev/k8s.io/klog#pkg-overview